src/common/interop/HotkeyManager.cpp

```diff
@@ -31,6 +31,13 @@ void HotkeyManager::KeyboardEventProc(KeyboardEvent ^ ev)
     if (hotkeys->ContainsKey(pressedKeysHandle))
     {
         hotkeys[pressedKeysHandle]->Invoke();
+
+        // After invoking the hotkey send a dummy key to prevent Start Menu from activating
+        INPUT dummyEvent[1] = {};
+        dummyEvent[0].type = INPUT_KEYBOARD;
+        dummyEvent[0].ki.wVk = 0xFF;
```

Comment:
In ShortcutGuide we use `0xCF` instead of `0xFF` which is better to use.

Reply:
`0xFF` doesn't appear in the virtual key code list https://docs.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes, whereas `0xCF` is used for some Xbox key which is why we used 0xFF. Haven't found any side effects of using 0xFF so far.

Reply:
I'll open an issue to change `0xCF` in the ShortcutGuide.